src/quo2/components/settings/category_list/view.cljs

```diff
@@ -0,0 +1,63 @@
+(ns quo2.components.settings.category-list.view
+  (:require [quo2.components.settings.category-list.style :as style]
+            [quo2.components.settings.settings-list.view :refer [badge-icon
+                                                                 browser-context-icon chevron-icon communities-icons right-button]]
+            [quo2.components.markdown.text :as text]
+            [react-native.core :as rn]))
+
+
+
+(defn category-title
+  [title]
+   (js/console.log "title: " title)
+  [rn/view
+   {:style style/title-container}
+   [text/text
+    {:accessibility-label :setting-item-name-text
+     :ellipsize-mode      :tail
+     :style               (style/title)
+     :number-of-lines     1
+     :weight              :medium
+     :size                :paragraph-1}
+    title]])
+
+(defn category-list
```

Comment:
We prefer to name the main component of any `view.cljs` file as just `view` to remove the redundancy of `category-list/category-list`.